diff --git a/src/lightning.py b/src/lightning.py
index 767a5b6..968daae 100644
--- a/src/lightning.py
+++ b/src/lightning.py
@@ -333,6 +333,7 @@ class MLP(Classifer):
         self.model.apply(self.init_weights)
 
     def forward(self, x):
+        print('Size: ', x.size())
         batch_size, channels, width, height = x.size()
         x = rearrange(x, 'b c w h -> b (w h c)')
         return self.model(x)
